recipes/create-ca-webapp-directories.rb

```diff
@@ -0,0 +1,15 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: create-python-webapps-directories
+
+[
+  "/home/web/sites",
+  "/home/web/sock",
+  "/home/web/logs"
```

Comment:
is the log rotation aggressive enough to ensure we won't fill the root partition?